packages/site/src/pages/api/content/preview.ts

```diff
@@ -0,0 +1,26 @@
+import { compileMDX } from '@jpmorganchase/mosaic-site-middleware';
```

Comment:
Not sure if this should be removed (as we don't need it) or if it should be kept and also added to the site generator template...

This the endpoint the preview component hits to compile the markdown pulled from the IBCE

thoughts? @lukeac123 @lilyvc @mark-tate @tomhazledine

Reply:
If it's not being used what's the reasoning to keep it ?